src/main/java/com/launchdarkly/client/LDConfig.java

```diff
@@ -335,6 +339,21 @@ public Builder startWaitMillis(long startWaitMillis) {
       return this;
     }
 
+   /**
+     * Enable event sampling. When set to the default of zero, sampling is disabled and all events
+     * are sent back to LaunchDarkly. When set to greater than zero, there is a 1 in
+     * <code>samplingInterval</code> chance events will be will be sent.
+     *
+     * <p>Example: if you want 5% sampling rate, set <code>samplingInterval</code> to 20.
+     *
+     * @param samplingInterval the sampling interval.
+     * @return the builder
+     */
+    public Builder samplingInterval(int samplingInterval) {
```

Comment:
I copied the semantics in the go client, but putting myself in the user's shoes I prefer using a percentage value here (and everywhere else) in the form of a floating point value from 0 to 1.0 where at 1.0 all events get sent and at 0.0 none get sent. I think consistency is more important, so we should leave it or change it everywhere.